### PR TITLE
ci: uses aliases for the branches

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

Use aliases and support current minor release branch and next minor release branch for the two major releases.

## Why is it important?

We don't need to worry about maintaining those schedule jobs anymore

## Issues

Requires https://github.com/elastic/apm-pipeline-library/pull/1488
Relies on https://github.com/elastic/apm-pipeline-library/blob/main/vars/bumpUtils.txt

### CODE REVIEW

File: .ci/schedule-daily.groovy [hunk 1]
```diff
@@ -37,7 +31,37 @@ pipeline {
...
+    if (!branches.contains(branchName)) {
+      branches << branchName
+    }
+  }
+
+  def quietPeriod = 0
+  branches.each { branch ->
+    build(quietPeriod: quietPeriod, job: "Beats/beats/${branch}", parameters: [booleanParam(name: 'macosTest', value: true)], wait: false, propagate: false)
+    // Increate the quiet period for the next iteration
+    quietPeriod += quietPeriodFactor
```
Comment (mdelapenya) [inline .ci/schedule-daily.groovy:48]: I think we are missing `quietPeriodFactor` defined or read from args

File: .ci/schedule-weekly.groovy [hunk 2]
```diff
@@ -37,7 +31,37 @@ pipeline {
...
+    if (!branches.contains(branchName)) {
+      branches << branchName
+    }
+  }
+
+  def quietPeriod = 0
+  branches.each { branch ->
+    build(quietPeriod: quietPeriod, job: "Beats/beats/${branch}", parameters: [booleanParam(name: 'awsCloudTests', value: true)], wait: false, propagate: false)
+    // Increate the quiet period for the next iteration
+    quietPeriod += quietPeriodFactor
```
Comment (mdelapenya) [inline .ci/schedule-weekly.groovy:48]: Ditto

File: .ci/schedule-daily.groovy (hunk 1 above)
Comment (v1v) [reply .ci/schedule-daily.groovy:48]: Good catch, it's `args.quietPeriodFactor`

```suggestion
    quietPeriod += args.quietPeriodFactor
```

File: .ci/schedule-weekly.groovy (hunk 2 above)
Comment (v1v) [reply .ci/schedule-weekly.groovy:48]:
```suggestion
    quietPeriod += args.quietPeriodFactor
```

**Review by mdelapenya — APPROVED** [review]

**Review by kuisathaverat — APPROVED** [review]

**Pull request merged**